src/AWS.Deploy.Orchestration/OptionSettingHandler.cs

```diff
@@ -333,5 +349,79 @@ public bool IsSummaryDisplayable(Recommendation recommendation, OptionSettingIte
 
             return true;
         }
+
+        /// <summary>
+        /// Checks whether the option setting item has been modified by the user. If it has been modified, then it will hold a non-default value
+        /// </summary>
+        /// <returns>true if the option setting item has been modified or false otherwise</returns>
+        public bool IsOptionSettingModified(Recommendation recommendation, OptionSettingItem optionSetting)
```

Comment:
I'm worried this yet another place that we have to remember to update whenever we add a new type to `OptionSettingValueType`. Could you add a comment to the `OptionSettingValueType` enum that list out these are the places that need to be changed if adding a new data type.

Reply:
Done